apps/api/src/app/topics/use-cases/create-topic.use-case.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { TopicRepository } from '@novu/dal';
+import { ObjectId } from 'mongoose';
```

Comment:
This import is not allowed to happen in the `API` level, we need to somehow re export it from the dal. Since the API doesn't have and should not have the mongoose dependcy :)